src/main/java/org/folio/bulkops/service/ProfileService.java

```diff
@@ -47,13 +48,23 @@ public ProfileSummaryResultsDto getProfileSummaries(String query, Integer offset
       )
     );
 
-    List<org.folio.bulkops.domain.dto.ProfileSummaryDTO> items = page.stream()
-      .map(profileMapper::toSummaryDTO)
+    List<ProfileSummaryDTO> items = page.stream()
```

Comment:
Can this be renamed ProfileSummaryDTO -> ProfileSummaryDto to follow common naming approach accepted in the module?